platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/maplayout/DebugModeActivity.java

```diff
@@ -47,9 +49,21 @@ protected void onCreate(Bundle savedInstanceState) {
       @Override
       public void onMapReady(@NonNull MapboxMap map) {
         mapboxMap = map;
+
+        mapboxMap.getUiSettings().setZoomControlsEnabled(true);
+
+        // show current zoom level on screen
+        final TextView textView = (TextView) findViewById(R.id.textZoom);
+        mapboxMap.setOnCameraChangeListener(new MapboxMap.OnCameraChangeListener() {
+          @Override
+          public void onCameraChange(CameraPosition position) {
+            textView.setText(String.format("Zoom : %s", position.zoom));
```

Comment:
Agreed, but you could always split it into more literals which opens the possibility of explaining your intention clearer.
BTW, as you mentioned putting it in `strings.xml` is also a great solution.